docs/ember/using-addons.md

```diff
@@ -51,24 +51,46 @@ If you are already using [strict mode] templates (via [first class component tem
 
 But more likely you are still using classic `.hbs` template files, for which Glint needs to know e.g. which component _name_ maps to which component _class_ and hence its type. This is managed by the [Template Registry], which needs to be extended for all the components, helpers and modifiers provided by the addon.
 
-By convention according to the [Authoring Guide][authoring], the addon will ship a `glint.d.ts`
-file that already contains all the necessary registry entries. You just need to import this file from somewhere in your app, e.g. where you already import `@glint/environment-ember-loose`:
+By convention according to the [Authoring Guide][authoring], the addon will ship a `template-registry.d.ts` file that exports a type containing all the registry entries for components, helpers and modifiers it exposes. To use these entries, you can declare that the `@glint/environment-ember-loose` registry `extends` the addon registry in the same place you add your local declarations.
 
 {% code title="types/global.d.ts" %}
 
 ```typescript
 import '@glint/environment-ember-loose';
-import 'ember-responsive-image/glint';
+
+import type ResponsiveImageRegistry from 'ember-responsive-image/template-registry';
```

Comment:
Just realized that I have (accidentally) plugged my own addon here. Ok for me of course, but also totally ok if we "anonymize" this as "awesome-addon" or something like this as above! :wink: